src/client/Root.js

```diff
@@ -9,27 +9,25 @@ import {ensureState} from 'redux-optimistic-ui';
 const lookConfig = Presets['react-dom'];
 lookConfig.styleElementId = '_look';
 
-process.env.NODE_ENV !== 'production' &&
+if (!__PRODUCTION__) {
   lookConfig.plugins.push(Plugins.friendlyClassName);
```

Comment:
By the way, I haven't seen this work in development mode. Not sure why. Not super worried about it, but it isn't interesting...